test/parallel/test-message-channel-sharedarraybuffer.js

```diff
@@ -0,0 +1,28 @@
+/*global SharedArrayBuffer*/
```

Comment:
Is it worth adding this to `.eslintrc.yaml` instead?

Reply:
Fwiw, I opened https://github.com/sindresorhus/globals/pull/119 so that at some point eslint picks this up itself